pages/documentation/grid/model/index.md

```diff
@@ -10,150 +10,122 @@ latex: true
 
-<span style="color:red"> TODO: when we have aliases, add a description here too.</span>
+To build an electrical network model using iIDM first the substations must be defined. The equipment in a substation (loads, generators, shunts, Static Var Compensators, DC converters ...) are grouped in voltage levels. Transformers present in a substation connect its different voltage levels. Transmission lines (AC and DC) connect the substations. A connection point for an equipment is a _terminal_.
+
+The PowSyBl grid model allows a full representation of the substation connectivity where all the switching devices and busbars are defined (_node/breaker_ level). Automated topology calculation permits to obtain views of the network up to the _bus/branch_ level.
+
+Different states of the network can be stored together with the power system model in an efficient way. The set of attributes that define a given state of the network (steady state hypothesis and state variables) are collectively organized in _variants_. The user can create and remove _variants_ as needed. Setting and getting variant dependent attributes on network objects use the current _variant_.
+
+A set of PowSyBl networks can be merged together in a single network view, and sub-parts of the network model can be easily extracted as separate networks.
+
+All elements modeled in the network are identified through a unique ID, and optionally described by a name that is easier to interpret for a human. All components can be _extended_ by the user to incorporate additional structured data.
 
 ## Network core model
 
-### Network 
+In the following sections the different network components are described in terms of its main attributes and electro-technical representation. The attributes shared by all the network components are described in the next table:
+
+| Attribute | Description |
+| --------- | ----------- |
+| $$Id$$ | Unique Id assigned to each network component |
+| $$Name$$ | Human readable alphanumeric identifier |
+| $$Fictitious$$ | To identify non-physical network components |
+| $$Aliases$$ | Additional unique identifiers associated with each network component |
+| $$Properties$$ | To add additional data items to network components |
+
+The `Id` is the only required attribute and by default `Fictitious` is set to `false`. `Aliases` offers the possibility of adding additional unique identifiers to each component. An `AliasType` can be specified to indicate what it corresponds to. `Properties` allows to associate additional arbitrary data items under the general schema of pairs `<key, value>`.
```

Comment:
We didn't want to document all the attributes of the model, but only the electrical characteristics. I don't know if it's a good idea or not.

Reply:
I would also change `AliasType` that is too "technical". I would explain that you can qualify each alias

Reply:
Done

Reply:
Indeed `AliasType` is maybe too technical, but we find a lot in `XIIDM` files. So it means that users can met some and it is useful to define what is an alias and why we use them in IIDM. But not in each equipment or class.